### PR TITLE
[SCRUM-34] feat: added docker file and docker compose file

### PR DESCRIPTION
**Pull request by Clue355**
- Added a dockerfile to admin_server to run the built server in a container
- Added a docker compose file to root folder to start both servers with one command
- uploaded built docker image for admin_server to AWS ERC

### CODE REVIEW

**Comment by allaboutmike** [comment]
Only the admin_server docker file is present. Are there others missing?

**Comment by allaboutmike** [comment]
Also, this dockerfile assumes that the build has already been run locally. Will we run the build as a part of the action? Or should we use a build image to build the backend within docker and then move the completed JAR into a deployment image afterwards?

**Comment by Clue355** [comment]
I'll edit it to build then run the admin_server.

**Comment by allaboutmike** [comment]
> I'll edit it to build then run the admin_server.

Use something like this: https://docs.docker.com/build/building/multi-stage/ to split the build stage and the run stage. That way the resulting image only has the jar, and not the source code.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+services:
+  admin_server:
+    build:
+      dockerfile: ./admin_server/Dockerfile
+    ports:
+      - "8080:8080"
+    environment:
+      - SPRING_PROFILE=production
```
Comment (allaboutmike) [inline docker-compose.yml:8]: This would be SPRING_PROFILES_ACTIVE.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  facial_recognition:
+    build:
+      dockerfile: ./facial_recognition/Dockerfile
+    ports:
+      - "9090:9090"
+    environment:
+      - PYTHON_PROFILE=production
```
Comment (allaboutmike) [inline docker-compose.yml:16]: There should be a Jira ticket to update these environment variables to what the flask application needs.

**Review by allaboutmike — DISMISSED** [review]
LGTM. Not sure if you want to setup the build Dockerfile in a separate task or this one.

**Review by allaboutmike — APPROVED** [review]
Excellent!

**Pull request merged**